### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "Cancelled"

### DISCUSSION THREAD

**Discussion by KronosWasTaken** [community/community#13738] [Copilot Conversations]
how to fix this
 i'm on manjaro kde

**Comment by davecheney** [comment]
Are you using VSCode? Copilot does not work with VSCodium or OSSCode.

**Reply by KronosWasTaken** [reply]
Yea i'm using the snap package of vscode
Does it not work with snap?

**Reply by davecheney** [reply]
Are you sure that's VSCode? Many Linux distros package VSCodium or OSSCode.

**Comment by KronosWasTaken** [comment]
Yea i'm sure that it's vscode i got it from the snapstore

**Comment by Alyxdev-xyz** [comment]
I am experiencing the same issue with VSCode, macOS 11.6.

However, using _nightly build_ instead bypassed the issue _(copilot v1.10.5413, nightly v1.10.5423)_.

**Reply by presn** [reply]
Same issue with VS Code Version: 1.66.0 (system setup)
Commit: e18005f0f1b33c29e81d732535d8c0e47cafb0b5
Date: 2022-03-30T05:50:14.623Z
Electron: 17.2.0
Chromium: 98.0.4758.109
Node.js: 16.13.0
V8: 9.8.177.11-electron.0
OS: Windows_NT x64 10.0.19042

Tried GitHub Copilot Nightly  v1.11.5497 and GitHub Copilot v1.11.5453 - both with no luck (GitHub Copilot could not connect to server. Extension activation failed: "Cancelled")

**Reply by davecheney** [reply]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by presn** [reply]
Trying at home, and neither isp nor goverment are intercepting SSL traffic.
Tried without any proxy. Writing this comment from same machine as VSCode.

Which host:port does it try to connect to?

**Reply by davecheney** [reply]
One of `https://api.github.com` or `https://copilot-proxy.githubusercontent.com`

**Reply by presn** [reply]
This is my nmap output for these two hosts:

Starting Nmap 7.92 ( https://nmap.org ) at 2022-04-04 10:54 W. Europe Daylight Time
Nmap scan report for api.github.com (140.82.121.6)
Host is up (0.017s latency).
rDNS record for 140.82.121.6: lb-140-82-121-6-fra.github.com

TRACEROUTE (using port 443/tcp)
HOP RTT      ADDRESS
-- 8<
9   ... 10
11  18.00 ms lb-140-82-121-6-fra.github.com (140.82.121.6)

Nmap done: 1 IP address (1 host up) scanned in 3.86 seconds



Starting Nmap 7.92 ( https://nmap.org ) at 2022-04-04 10:56 W. Europe Daylight Time
Nmap scan report for copilot-proxy.githubusercontent.com (20.81.50.155)
Host is up (0.10s latency).

TRACEROUTE (using port 443/tcp)
HOP RTT       ADDRESS
-- 8<
13  ... 22

23  102.00 ms 20.81.50.155

Nmap done: 1 IP address (1 host up) scanned in 6.70 seconds


Looks like these hosts are reachable from my machine.

What debugging step shall I take next?

**Reply by presn** [reply]
Turned out to be a browser issue on my machine.
ty
